### ISSUE TITLE
[Soviet+Misty(?)] Opened a Soviet Building, and saw command blocks

### ISSUE THREAD

**Issue by DocFortuna** [JoelZBub/organic-free-range-pack#82]
### Describe the Bug:
My friends and i found soviet bunkers in the misty world, We acquired the wrench and opened it up, only us to see redstone, a command block and an entity detector, with the rest of the building inaccessible.

### How Can We Reproduce the Bug:
1. Get wrench and open hermetic doors/hatch
2.  Proceed some ways into building
3.
4. 

More steps...

### Expected behavior:
The red block next to the command block had a button in its UI called Players, I'm assuming it was supposed to detect players entering the soviet buildings and make a special event happen?

### If Applicable, Add Screenshots or Video to Explain the Bug:
![2021-03-08_21 01 06](https://user-images.githubusercontent.com/80305776/110421626-bce91e80-8052-11eb-9594-3ae1262b487d.png)
![2021-03-08_21 01 28](https://user-images.githubusercontent.com/80305776/110421628-beb2e200-8052-11eb-8cc5-61a83c8ab50a.png)
![2021-03-08_21 01 45](https://user-images.githubusercontent.com/80305776/110421633-bfe40f00-8052-11eb-8e5f-0ecde1c59495.png)

### Crash Report or Game Log
Paste your crash report or log file to https://gist.github.com/ and paste the link here.

### Please complete the following information:
 - Have you added or removed any mods, and if so, which?: None
 - Modpack Version Number :1.2.0
 - Is OptiFine installed? : No
 - Does this issue affect Server, Client, or Both? :

**Comment by JoelZBub** [comment]
You need to enable command blocks on the server.properties file.
The correct file with all the necessary settings to make the server operate correctly is included with the server download on Curseforge.

**Comment by JoelZBub** [comment]
Tl;dr : Use the server.properties that comes with the server download.
